README.md

```diff
@@ -94,17 +85,16 @@ Read our [announcement on Hybrid Execution] for more information.
 
 1. Install the official [GlareDB Python library]
 
-     ```shell
-     pip install glaredb
-     ```
+   ```shell
+   pip install glaredb
+   ```
 
 2. Import and use `glaredb`.
 
-     ```python
-     import glaredb
-     con = glaredb.connect()
-     con.sql("select 'hello world';").show()
-     ```
+   ```python
+   import glaredb
+   glaredb.sql("select 'hello world';").show()
```

Comment:
I think I'd inline/chain a connect method: without this we don't establish that you can connect to cloud, and when people start doing stateful things, not being bound to a connection object is confusing.